src/pages/items/[itemId]/edit.svelte

```diff
@@ -0,0 +1,57 @@
+<script>
+import user from '../../../authn/user'
+import { Breadcrumb, ItemForm } from '../../../components'
+import { loading } from '../../../components/progress'
+import { itemsByPolicyId, loadItems, updateItem } from '../../../data/items.js'
+import { goto } from '@roxi/routify'
+import { Page } from '@silintl/ui-components'
+
+export let itemId
+
+const breadcrumbLinks = [
+  {
+    name: "Items",
+    url: "/items",
+  },
+  // TODO: make this fetch the name of the item and have that as the name 
+  {
+    name: "This Item",
+    url: `/items/${itemId}`
+  },
+  {
+    name: "Edit",
+    url: `/items/${itemId}/edit`
+  }
+]
+
+$: policyId = $user.policy_id
+$: policyId && loadItems(policyId)
+$: items = $itemsByPolicyId[policyId] || []
+$: item = items.find(anItem => anItem.id === itemId) || {}
```

Comment:
```suggestion
$: item = items.find(theItem => theItem.id === itemId) || {}
```
It's not just "any" item. :-)